pnpm-lock.yaml

```diff
@@ -400,7 +400,7 @@ importers:
       rollup-plugin-node-polyfills: 0.2.1
       rollup-plugin-serve: 2.0.2
       sass: 1.58.0
-      vite: 4.1.1
+      vite: 4.1.1_sass@1.58.0
```

Comment:
This is unrelated, but happens for me as well when I do `pnpm install` on a clean master. So it's fine I guess 🤷